.github/workflows/test-r-scripts.yaml

```diff
@@ -63,13 +76,27 @@ jobs:
           r-version: ${{ matrix.config.r }}
           http-user-agent: ${{ matrix.config.http-user-agent }}
 
+      - name: Install system dependency of usethis >= 2.0.0
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt install libgit2-dev
+
       - name: Install dependencies from CRAN
         run: |
           install.packages("rmarkdown")
           install.packages("knitr")
           install.packages("renv")
-          dependencies <- renv::dependencies()$Package
-          install.packages(dependencies)
+          deps <- unique(renv::dependencies("../PACTA_analysis")$Package)
+          exclude <- c(
+            "R",
+            "utils",
+            "base",
+            "tools",
+            "create_interactive_report",
+            "PACTA.analysis",
+            "StressTestingModelDev"
+          )
```

Comment:
We are now installing only what we need, and avoiding avoidable installation problems.